Building uppy.io/docs VS uppy.io/blog (#243)
transloadit/uppy.io: README.md

# Uppy.io

> Website and documentation for [uppy.io](https://uppy.io).

Reviewer: Well the documentation was moved to the other repo
```suggestion
The official [Uppy](https://uppy.io) website.
```

Code for [uppy.io/blog](https://uppy.io/blog) lives in this repo.  
Code for [uppy.io/docs](https://uppy.io/docs/quick-start) lives in

> [github.com/transloadit/uppy](https://github.com/transloadit/uppy).

Reviewer: ```suggestion
[transloadit/uppy](https://github.com/transloadit/uppy).
```

Author: Why? I feel `github.com` prefix makes it more intelligible, and removes the need to remember github-specific shortcuts.

## Developer guide

If you want to edit [uppy.io/blog](https://uppy.io/blog), just edit the files in
this repo.

If you want to edit [uppy.io/docs](https://uppy.io/docs/quick-start), then you
should:

> 1. Clone both the **uppy** repo and **uppy.io** repo.

Reviewer: ```suggestion
1. Clone both the `transloadit/uppy` and `transloadit/uppy.io` repositories.
```

> 2. Then, from your **/uppy.io** repo, run:

Reviewer: ```suggestion
2. Then, from your `uppy.io` clone, run:
```

   ```sh
   ln -s /path-to-your-uppy-repo/docs docs
   ```

>    Now, treat **/uppy/docs** as your working directory, but run the build/lint
>    scripts from your **/uppy.io** directory - **/uppy.io** will pick up your
>    changes.

Reviewer: I don't really understand the "treat as your working directory"
```suggestion
   Now, when running the build/lint scripts from the root of your `transloadit/uppy.io` clone, it will pick up your local changes on the `docs/` folder. Note that changes in the `docs/` changes should be submitted to the `transloadit/uppy` repository.
```

Author: "treat /uppy/docs as your working directory" means "/uppy/docs is a folder where you will be editing files, then committing the files, and then pushing the files".

corepack yarn dev
```

This command builds **/uppy/docs** and **/uppy.io**, and shows them in a

> browser. Most changes are reflected live without having to restart the server.

Reviewer: I think the previous sentence was clearer. Can you explain what you don't like about it?

### Lint

```sh
corepack yarn run format
corepack yarn lint
```

> This command lints **/uppy/docs** and **/uppy.io**.

Reviewer: ```suggestion
N.B.: These commands will also report lint errors from the `docs` folder that's hosted on the `transloadit/uppy` repository.
```

Author: My version is more consistent with the phrasing of other command descriptions.

This command generates static content into the `build` directory and can be
served using any static contents hosting service.

> 
> ---
> 
> This website is built using [Docusaurus 2](https://docusaurus.io/), a modern
> static website generator.

Reviewer: We might want to simply remove this – we're now using Docosaurus v3, and it seems pointless to put that (ever so small) maintenance burden on us when the information is easily findable in the `package.json`.
```suggestion

```
